apps/web/server/utils/sync.ts

```diff
@@ -1,32 +1,20 @@
+import ky from "ky";
+
 const SYNC_URL = process.env.SYNC_URL || "http://localhost:5000";
 const SYNC_KEY = process.env.SYNC_KEY || "verysecurekey";
 
 export async function indicateSettingsSync(channel: number): Promise<void> {
-  const url = `${SYNC_URL}/settings/sync/${channel}`;
-
-  await fetch(url, {
-    method: "POST",
-    body: `{"key": "${SYNC_KEY}"}`,
-  });
+  const url = `${SYNC_URL}/api/v1/unsafe/${channel}/settings/sync`;
+  await ky.post(url, { json: { key: SYNC_KEY } });
```

Comment:
We spoke on this topic, but I don't remember the final thing... `ky` doesn't trigger `exception` on status `>400` right?